x/ecocredit/base/client/tx.go

```diff
@@ -109,48 +113,77 @@ regen tx ecocredit create-class regen1elq7ys34gpkj3jyvqee0h6yk4h9wsfxmgqelsw,reg
 
 // TxCreateProjectCmd returns a transaction command that creates a new project.
 func TxCreateProjectCmd() *cobra.Command {
+	var classID, referenceID, projectFee string
+
 	cmd := &cobra.Command{
-		Use:   "create-project [class-id] [jurisdiction] [metadata] [flags]",
-		Short: "Create a new project within a credit class",
-		Long: `Create a new project within a credit class.
+		Use:   "create-project [jurisdiction] [metadata] [flags]",
+		Short: "Create a new project",
+		Long: `Create a new project. By default, this creates an unregistered project that is not enrolled in any credit
+class. If the class flag is provided, the signer must be an issuer of the class and the project will be enrolled in
+the class automatically.
```

Comment:
Just took a look at the tests, and it looks like an integration test is failing due to changes from this command `regen tx ecocredit create-project`. The class-id should be passed as a flag now.

Test failure: https://github.com/regen-network/regen-ledger/actions/runs/9872434922/job/27262543729#step:5:216

Integration test: https://github.com/regen-network/regen-ledger/blob/3d818cf6e01af92eed25de5c17728a79070f56a3/x/ecocredit/client/testsuite/suite.go#L380-L390